src/chains/types.ts

```diff
@@ -25,7 +49,23 @@ export type Precompile = {
   references: string[];
 };
 
+export type Opcode = {
+  number: number;
+  name: string;
+  description: string;
+  minGas: number;
+  gasComputation?: GasComputation;
+  inputs: Variable[];
+  outputs: Variable[];
```

Comment:
Should we add `| undefined` here, since some opcodes have no inputs (e.g. NUMBER) and some have no outputs (e.g. MSTORE)? This way it's a bit more explicit. Similar for the `Example` type

Reply:
Instead of adding `| undefined`, i made the fields optional `inputs?: Variables[]`. This way, we don't need to specify empty fields in the opcode configurations. See [7312417](https://github.com/mds1/evm-diff/pull/12/commits/7312417fce2e042769b269047126d09b35e8c6e1)